data/interface.go

```diff
@@ -225,7 +225,7 @@ type StorageManager interface {
 
 // TrieFactory creates new tries
 type TrieFactory interface {
-	Create(config.StorageConfig, string, bool, uint) (StorageManager, Trie, error)
+	Create(config.StorageConfig, string, bool, bool, uint) (StorageManager, Trie, error)
```

Comment:
let's switch to argument parameter here :)

Reply:
Done.